rapid7_insight_agent/help.md

```diff
@@ -0,0 +1,167 @@
+# Description
+
+Take actions on an Insight Agent
```

Comment:
Improve this with more details about the Insight Agent, there's existing docs we can pull from.

Have links too